src/FormBuilder.php

```diff
@@ -587,6 +588,10 @@ private function _buildAttrs(array $props = [], array $ignore = []): string
             $props['aria-describedby'] = $this->_getIdHelp();
         }
 
+        if($this->_required === true) {
+            $props['required'] = "required";
+        }
+
```

Comment:
@netojose , can you check if this implementation is correct? I'm facing weird bugs with required attribute at my forms.  #38 

```php
// the code below is ok
{!! Bootstrap4::text('firstname', 'Name') !!} // not required
{!! Bootstrap4::text('lastname', 'Lastname') !!} // not required

// the code below is not rendering fields correctly
{!! Bootstrap4::text('firstname', 'Name')->required() !!} // required
{!! Bootstrap4::text('lastname', 'Lastname') !!} // required (I was expecting not required)

// this is ok:
{!! Bootstrap4::text('firstname', 'Name') !!} // not required
{!! Bootstrap4::text('lastname', 'Lastname')->required() !!} // required

// but this is not:
{!! Bootstrap4::text('firstname', 'Name') !!} // not required
{!! Bootstrap4::text('lastname', 'Lastname')->required() !!} // required
{!! Bootstrap4::text('other_field', 'Other') !!} // required (I was expecting not required)
```

Reply:
@ricardovigatti , I will see this tomorrow (jan-4). I already added a reminder here. And I will check other opened issues.
Thanks for everything!

Reply:
After some tests, I can confirm that this behavior started after v2.0.4.

Reply:
After more tests, I realize that the problem is not issued if I set the "required" attribute through **attrs()** method. But the implementation of this method not seems to be right. See:

```php
{!! Bootstrap4::text('name1', 'Name 1')->attrs(['required' => true]) !!} // ok, required
{!! Bootstrap4::text('name2', 'Name 2')->attrs(['required' => false]) !!} // not ok, still required
{!! Bootstrap4::text('name3', 'Name 3')->attrs(['required' => null]) !!} // ok, not required
```